spec/controllers/curation_concerns/generic_works_controller_spec.rb

```diff
@@ -4,7 +4,16 @@
 # which is included into .internal_test_app/app/controllers/generic_works_controller.rb
 describe CurationConcerns::GenericWorksController do
   let(:user) { create(:user) }
-  before { sign_in user }
+  let(:work) { create(:generic_work, user: user) }
+  let(:sipity_entity) do
```

Comment:
It's better to use a before block though. Otherwise rubocop rspec will complain.